game-core/src/test/java/games/strategy/triplea/delegate/battle/steps/fire/firststrike/ClearFirstStrikeCasualtiesTest.java

```diff
@@ -37,134 +36,118 @@
 
   @ParameterizedTest
   @MethodSource
-  void getStep(
-      final List<BattleStateVariation> parameters,
-      final boolean willExecute,
```

Comment:
While updating this test for the boolean -> side change, I noticed that `willExecute` isn't actually needed so I removed it as part of this PR.